docs/modules/generic.rst

```diff
@@ -7,12 +7,19 @@ Generic PDF objects
     :show-inheritance:
     :exclude-members: Destination, Field, Fit, RectangleObject
 
+
 .. autoclass:: pypdf._protocols.PdfObjectProtocol
     :members:
     :undoc-members:
     :show-inheritance:
 
 
+.. autoclass:: pypdf._protocols.PdfCommonDocProtocol
```

Comment:
We might want to move the protocols to the new `PdfDocCommon` page, or do you think that it does not belong there?

Reply:
the protocols are for me of no interest : we have to refer to the implementations PdfReader/PdfWriter ; even when  PdfCommonDoc  is called it shall be considered to use a PdfReader or PdfWriter

Reply:
These are required for the cross-references. I am just not sure whether they might be moved to another documentation file/page.